### PR TITLE
Update borehole-status-07.06.2023.ttl

### PR DESCRIPTION
**Pull request by johnmckellar**
@Steven-Bowden @LizDerrington @GSQ-AI 

Nick, for your assessment please before I include others in on the PR
Cheers
John

### CODE REVIEW

**Review by nicholascar — APPROVED** [review]
Changes look fine to me - these are SME content additions. Vocab valid according to validator after changes.

**Comment by johnmckellar** [comment]
@LizDerrington @GSQ-AI @Steven-Bowden @nicholascar 
Liz, Andrew, Please review. Nick has validated the file, and made a minor format change, as I also did this morning. Cheers

File: vocabularies-gsq/borehole-status.ttl [hunk 1]
```diff
@@ -12,22 +12,23 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
 :proposed
     a skos:Concept ;
+  dcterms:references "Geoscience Australia, 2021: Borehole (Well) Status Vocabulary. <https://db-content.ga.gov.au/data_dictionary/Borehole_Status_vocabulary_2021.pdf>"@en ;
     skos:prefLabel "Proposed"@en ;
-    skos:definition "A concept or application exists to drill a borehole, but drilling has not yet commenced or been approved"@en ;
-    dcterms:provenance "Modified from: Geoscience Australia, modified from PPDM Wellbore Status vocabulary, 2014; CoalLog. Similar to PPDM's 'location' classification" ;
-    skos:usageNote "Used by GA, QLD, BOM" ;
+    skos:definition "A (pre-application) concept/intention or application exists to drill a borehole, but drilling has not yet been approved."@en ;
```
Comment (GSQ-AI) [inline vocabularies-gsq/borehole-status.ttl:17]: Proposed status is a notice of intention to drill and is separate from all Landholder approvals for access and activity. I think that this definition is deviating from the actual purpose of "proposed" in Queensland Government systems and introduces ambiguity by introducing 'pre-application' into the language.

Simplifying this to something like "An intention exists to drill a borehole, but drilling has not yet commenced." would perhaps not only be more succinct, but also clearer in terms of the actual definition.

File: vocabularies-gsq/borehole-status.ttl [hunk 2]
```diff
@@ -12,22 +12,23 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
...
+    dcterms:provenance "Modified from Geoscience Australia (2021)." ;
+    skos:scopeNote "Similar to usage by GA & QLD. Also, (1) cf. VIC (rephrased without change of meaning): ‘Well status for a proposed well, i.e., for a well that has not been drilled or for which a WCR, providing well co-ordinates, has not been received; (2) cf. WA (GSWA Borehole Status): Suggesting ‘planned’ for intention prior to application, and ‘proposed’ for a submitted application that has yet to be approved." ;
```
Comment (GSQ-AI) [inline vocabularies-gsq/borehole-status.ttl:19]: Why are we saying that the QLD definition is similar to usage by Queensland?

**Review by GSQ-AI — APPROVED** [review]
I was a little confused as to the purpose of this, and then realised after deeper review, that this is the harmonisation vocab. There's no issue with the harmonisation vocab differing from the QLD Site Status vocab, and I'm happy for this to go ahead for use in the harmonisation space. 

Please ignore the comments I made earlier, as they were made under the impression that this was for internal Queensland Government use. I would not be averse to making it clearer somehow that this is not a QGOV/GSQ vocab and is for the harmonisation project, especially given that we already have a Borehole Status collection. 

Given that I am not privy to the discussions of the Harmonisation project, I cannot really comment on the detail of these changes.

**Review by Steven-Bowden — APPROVED** [review]
Approved as part of the cross-jurisdictional harmonisation work being carried out. As it is a collaborative effort, I expect further changes and refinements to be made.

**Comment by johnmckellar** [comment]
Thanks guys. This will be a start to dealing more intensively with it at a jurisdictional level, including provision of a better definition for the vocab.

**Pull request merged**